index.js

```diff
@@ -110,6 +110,12 @@ Browserify.prototype.require = function (file, opts) {
     }
     
     if (!opts) opts = {};
+
+    // Prevent .require() from adding transforms. See: module-deps and
+    // https://github.com/substack/node-browserify/pull/1057
+    delete file.transform;
```

Comment:
You're mutating the object that was passed into `require`. This should be done after the `xtend` below.